easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,92 @@
+    ('locket', '0.2.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/l/locket'],
+        'checksums': ['1fee63c1153db602b50154684f5725564e63a0f6d09366a1cb13dffcec179fb4'],
+    }),
+    ('partd', '1.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/partd'],
+        'checksums': ['54fd91bc3b9c38159c790cd16950dbca6b019a2ead4c51dee4f9efc884f8ce0e'],
+    }),
+    ('HeapDict', '1.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/h/HeapDict'],
+        'checksums': ['40c9e3680616cfdf942f77429a3a9e0a76f31ce965d62f4ffbe63a83a5ef1b5a'],
+    }),
+    ('zict', '1.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/z/zict'],
+        'checksums': ['e34dd25ea97def518fb4c77f2c27078f3a7d6c965b0a3ac8fe5bdb0a8011a310'],
+    }),
+    ('tblib', '1.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tblib'],
+        'checksums': ['bd1ad564564a158ff62c290687f3db446038f9ac11a0bf6892712e3601af3bcd'],
+    }),
+    ('sortedcontainers', '2.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/sortedcontainers'],
+        'checksums': ['974e9a32f56b17c1bac2aebd9dcf197f3eb9cd30553c5852a3187ad162e1a03a'],
+    }),
+    ('msgpack', '0.6.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/msgpack'],
+        'checksums': ['4008c72f5ef2b7936447dcb83db41d97e9791c83221be13d5e19db0796df1972'],
+    }),
+    ('cloudpickle', '1.2.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/cloudpickle'],
+        'checksums': ['603244e0f552b72a267d47a7d9b347b27a3430f58a0536037a290e7e0e212ecf'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/d/dask'],
+        'checksums': ['63a0f77366108cdbf9350cf368042d6f5d8c9df581b9b17781f0a603e23ead6c'],
+    }),
+    ('distributed', '2.3.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/distributed'],
+        'checksums': ['94b7c494abfecad0b1fcd325365f6c804c2a6ea9ffc510bb9f368d33be338b96'],
+    }),
+    ('dask-mpi', '2.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/dask-mpi'],
+        'checksums': ['774cd2d69e5f7154e1fa133c22498062edd31507ffa2ea19f4ab4d8975c27bc3'],
+    }),
+    ('docrep', '0.2.7', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/docrep'],
+        'checksums': ['c48939ae14d79172839a5bbaf5a570add47f6cc44d2c18f6b1fac8f1c38dec4d'],
+    }),
+    ('dask-jobqueue', '0.6.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/dask-jobqueue'],
+        'checksums': ['b58a3ece521cf541bf90b81f89e9ed478d7cdea4bbe70b56a6cfedbd3b360209'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/dask-%s' % x for x in ['mpi', 'remote', 'scheduler', 'ssh', 'submit', 'worker']],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
You can drop this one when using `PythonBundle`

Reply:
done